### ISSUE TITLE
CoreDNS cannot access external domain names.

### ISSUE THREAD

**Issue by zchunhai** [kairen/kube-ansible#115]
I install the cluster with default parameters, but coredns not working properly.

Node OS: ubuntu 18.04

```
kubectl logs coredns-7945bc8d5c-qp5s7 -n kube-system --tail=100 -f

......
2018/08/20 07:02:44 [ERROR] 2 smtp.office365.com. A: unreachable backend: read udp 10.244.2.27:34504->10.96.0.10:53: i/o timeout
2018/08/20 07:02:44 [ERROR] 2 smtp.office365.com. AAAA: unreachable backend: read udp 10.244.2.27:40291->10.96.0.10:53: i/o timeout
2018/08/20 07:02:44 [ERROR] 2 smtp.office365.com. AAAA: unreachable backend: read udp 10.244.2.27:42891->10.96.0.10:53: i/o timeout
2018/08/20 07:02:44 [ERROR] 2 elasticsearch-logging. AAAA: unreachable backend: read udp 10.244.2.27:55194->10.96.0.10:53: i/o timeout
2018/08/20 07:02:44 [ERROR] 2 smtp.office365.com. AAAA: unreachable backend: read udp 10.244.2.27:36631->10.96.0.10:53: i/o timeout
......
```

```
kubectl describe pod coredns-7945bc8d5c-qp5s7 -n kube-system

......
Events:
  Type     Reason     Age               From               Message
  ----     ------     ----              ----               -------
  Normal   Scheduled  55m               default-scheduler  Successfully assigned kube-system/coredns-7945bc8d5c-qp5s7 to k8s-n2
  Normal   Pulled     54m               kubelet, k8s-n2    Container image "192.168.21.29:5000/coredns:1.1.3" already present on machine
  Normal   Created    54m               kubelet, k8s-n2    Created container
  Normal   Started    54m               kubelet, k8s-n2    Started container
  Warning  Unhealthy  4m (x9 over 51m)  kubelet, k8s-n2    Liveness probe failed: Get http://10.244.2.27:8080/health: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
```

**Comment by kairen** [comment]
@zchunhai Thanks you for your report, I'll try to reproduce it.

**Comment by kairen** [comment]
Fixed #130
